src/main/java/org/schemaspy/cli/CommandLineArguments.java

```diff
@@ -102,6 +102,15 @@ public class CommandLineArguments {
     )
     private boolean noImplied = false;
 
+    @Parameter(
```

Comment:
Is there a reason why this is added to CommandLineArguments instead of DotConfig?

Reply:
Orphans is a SchemaSpy concept, not a `dot` one, similar to how `--no-implied` is in `CommandLineArguments` while `-rankdirbug` ([link](https://graphviz.org/docs/attrs/rankdir/)) is in `DotConfig`.

Expressed differently, the first relates to our algorithms for what parts of the database we should display while the second is how the graph itself should be rendered/presented.

Reply:
We have Dot concept in SchemaSpy and the configuration is for that.

Configuration would be added to `org.schemaspy.output.dot.DotConfig` it would be used in `org.schemaspy.output.dot.schemaspy.DotFormatter`

Reply:
I've created issue #1252 to continue the discussion. How about we don't let this PR grow stale waiting for a decision? Even in the worst case, it's better to merge and then submit a new change.

Reply:
Yeah, sure go ahead.

My thinking is that both `no implied` and `rankdir` both control content of dot syntax file and nothing else in the system. (Might be wrong about no implied. But for this setting it only effects dot file content it doesn't remove orphans from the complete system. So over exposing it or lack of context, it's global and you need to look up the resource key or check other usages to find out.

I'm unsure about no implied since it's in this file.